package com.fabric.auth.exceptions;

public class UserAlreadyExistsException {
}
